blocks/init/src/Blocks/components/modal/assets/index.js

```diff
@@ -0,0 +1,24 @@
+import domReady from '@wordpress/dom-ready';
+import manifest from './../manifest.json';
+import MicroModal from 'micromodal';
+
+domReady(() => {
+	const selector = `.${manifest.componentJsClass}`;
+	const elements = document.querySelectorAll(selector);
+	const body = document.getElementsByTagName('body')[0];
+
+	// Append all modals at the bottom of body.
+	[...elements].forEach((element) => {
+		body.append(element);
+	});
```

Comment:
I told him to do this. This is because you can for example use modal to show video from the carousel and due to z-indexes and positions all can break. So if you transfer all modals to top level Dom you are ok

Reply:
It's basically to avoid styling issues and to act independently from block (even if they are part of if).